roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import AnsibleDockerClient
+
+
+def main():
+    client = AnsibleDockerClient(
+        argument_spec=dict(
+            name=dict(type='list'),
+        ),
+    )
+
+    client.module.exit_json(
+        changed=False,
+        info=client.info(),
+    )
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
hm, maybe it would be better to rename the module from `docker_info` to simply `docker`? That way, it makes sense for it to return not just info, but docker client info.